File: EticaretJava/app/src/main/java/com/example/eticaretjava/model/ProductListPage.java
package com.example.eticaretjava.model;


import com.google.gson.annotations.SerializedName;
import java.util.List;
import com.example.eticaretkotlin.model.Product.ProductListDto;

public class ProductListPage {

    @SerializedName("page")
    public int page;

    @SerializedName("per")
    public int per;

    @SerializedName("total")
    public int total;

    @SerializedName("items")
    public List<ProductListDto> items;
}

File: Eticaretjetpack/app/src/main/java/com/example/eticaretjetpack/model/ProductListPage.kt
package com.example.eticaretjetpack.model

import com.google.gson.annotations.SerializedName

data class ProductListPage(
    @SerializedName("page")
    val page: Int,

    @SerializedName("per")
    val per: Int,

    @SerializedName("total")
    val total: Int,

    @SerializedName("items")
    val items: List<ProductListDto>
)
